cocotb/handle.py

```diff
@@ -72,6 +72,8 @@ class SimHandleBase(object):
 
     def __init__(self, handle, path):
         """
+        .. Constructor. This RST comment works around sphinx-doc/sphinx#6885
+
```

Comment:
Rendering fixed by adding this line. Sphinx otherwise strips spaces for inherited (but not direct) docstrings, which breaks autodoc

By adding a comment here, the stripped spaces don't matter.